pybossa/jobs.py

```diff
@@ -149,9 +164,11 @@ def get_periodic_jobs(queue):
     leaderboard_jobs = get_leaderboard_jobs() if queue == 'super' else []
     weekly_update_jobs = get_weekly_stats_update_projects() if queue == 'low' else []
     failed_jobs = get_maintenance_jobs() if queue == 'maintenance' else []
+    completed_tasks_cleanup_job = get_completed_tasks_cleaup_jobs() if queue == "weekly" else []
```

Comment:
Typo on the method name `get_completed_tasks_cleaup_jobs() ` => `get_completed_tasks_cleanup_jobs()`
*minor*